frontend/src/component/common/FormTemplate/FormTemplate.tsx

```diff
@@ -165,22 +165,44 @@ const FormTemplate: React.FC<ICreateProps> = ({
     const { setToastData } = useToast();
     const smallScreen = useMediaQuery(`(max-width:${1099}px)`);
     const copyCommand = () => {
-        if (copy(formatApiCode())) {
-            setToastData({
-                title: 'Successfully copied the command',
-                text: 'The command should now be automatically copied to your clipboard',
-                autoHideDuration: 6000,
-                type: 'success',
-                show: true,
-            });
-        } else {
-            setToastData({
-                title: 'Could not copy the command',
-                text: 'Sorry, but we could not copy the command.',
-                autoHideDuration: 6000,
-                type: 'error',
-                show: true,
-            });
+        if (formatApiCode !== undefined) {
+            console.log('format');
+            if (copy(formatApiCode())) {
+                setToastData({
+                    title: 'Successfully copied the command',
+                    text: 'The command should now be automatically copied to your clipboard',
+                    autoHideDuration: 6000,
+                    type: 'success',
+                    show: true,
+                });
+            } else {
+                setToastData({
+                    title: 'Could not copy the command',
+                    text: 'Sorry, but we could not copy the command.',
+                    autoHideDuration: 6000,
+                    type: 'error',
+                    show: true,
+                });
+            }
+        }
+    };
+
+    const renderApiInfo = (apiDisabled: boolean) => {
+        if (!apiDisabled) {
+            return (
+                <>
+                    <StyledSidebarDivider />
+                    <StyledSubtitle>
+                        API Command{' '}
+                        <Tooltip title="Copy command" arrow>
+                            <IconButton onClick={copyCommand} size="large">
+                                <StyledIcon />
+                            </IconButton>
+                        </Tooltip>
+                    </StyledSubtitle>
+                    <Codebox text={formatApiCode!()} />{' '}
+                </>
+            );
```

Comment:
This will surely work just fine for now, but is there a world wherein the `apiDisabled` variable gets somehow uncoupled from `formatApiCode`, so that the function tries to force `formatApiCode` when it's undefined? Would it be better to just check for that value directly within the component?

Something like:
```tsx
if (!formatApiCode) return null
else return fullcomponent
```

Absolutely not a blocker, though! Just a thought I had.